R/create_citation.R

```diff
@@ -7,12 +7,15 @@
 #' with `asar`.
 #' @export
 #'
-#' @examples create_citation(
-#'   title = "SA Report for Jellyfish",
+#' @examples
```

Comment:
I think this might be another example that can be run if we want to bc all it does is create a string formed for cat. I would name the function something like `citation <- create_citation()` then end the example with cat(citation) so users can understand the use for the fxn

Reply:
👍 Edited as per your suggestion.